src/main/java/net/mcreator/generator/Generator.java

```diff
@@ -477,14 +511,13 @@ public List<GeneratorTemplate> getGlobalTemplatesList(@Nullable Map<?, ?> map, b
 				if (TemplateExpressionParser.shouldSkipTemplateBasedOnCondition(this, conditionRaw,
 						workspace.getWorkspaceInfo(), operator)) {
 					if (((Map<?, ?>) template).get("deleteWhenConditionFalse") != null && performFSTasks)
-						if (workspace.getFolderManager().isFileInWorkspace(new File(name))) {

```

Comment:
These changes are not related to this PR at all and reduce code readability. Revert all changes not directly related to the functionality of this PR. Marking as draft and keep it as draft until this is not resolved

Reply:
:heavy_check_mark: Fixed
>Marking as draft and keep it as draft until this is not resolved

Feels like I shouldn't touch it here again ~(who knows what else like those changes I was stupid enough for?)~.

Reply:
>Marking as draft and keep it as draft until this is not resolved

Does this require your confirmation, @KlemenDEV?

Reply:
Yes

Reply:
:heavy_check_mark: Fixed